services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str | None = None,
+    first_name: str | None = None,
+    last_name: str | None = None,
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> QuerySet[User]:
```

Comment:
The return type hint `QuerySet[User]` is incorrect for the `get_user` function. Since `get()` returns a single instance, the correct return type should be `User`.